packages/app/src/components/Visualization/StartScreen.js

```diff
@@ -24,20 +24,27 @@ const StartScreen = ({ error }) => {
                 engine,
                 6
             )
-            const visualizations = mostViewedVisualizationsResult.visualization
+            const visualizations = mostViewedVisualizationsResult.visualization // {position: int, views: int, id: string, created: string}
```

Comment:
Remove comments on lines 27 and 34

Reply:
I thought the comments were actually quite useful for future reference, to know the signature of the response. We don't do this anywhere else though, so I can remove them if you prefer.
Could prefix them with `Response: ` for clarification if that's better though.